packages/adapter-edgedb/tsconfig.eslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["tests", "src"],
```

Comment:
This file isn't needed anymore, could you remove it? 🙌 We only use one `tsconfig.json` file for each package now